feature-libs/organization/administration/core/http-interceptors/bad-request/bad-request.handler.ts

```diff
@@ -54,7 +56,7 @@ export class OrganizationBadRequestHandler extends HttpErrorHandler {
         this.unitMask,
         this.permissionMask,
         this.unknownMask,
-      ].some((mask) => mask.test(error.message))
+      ].some((mask) => mask.test(error.message ?? ''))
     );
   }
 
```

Comment:
method `protected handleOrganizationConflict` is missing a return type `: void`.  Since we're making sure all types are tidy and strict....